README.md

```diff
@@ -124,3 +124,15 @@ Not right now. We might add this feature in future, but for now, consider either
 
 - Adding your dependency to the package set if possible, or
 - Creating your own custom package set
+
+### Why are my changes not updated in my package set?
+
+Package sets are cached based on a git reference (e.g. tag or branch)
```

Comment:
Just to be clear, this only happens if you're using a branch, or something like master, right? The typical usage should be with _tags_, where this shouldn't happen. Maybe we should spell that out.

Reply:
You _can_ delete tags. Not good practice, but it happens and would create the same problem.

Reply:
Yes, I was using a branch, while figuring out how to create a package set and then tagged it once things worked.